### ISSUE TITLE
Bug: Report is overlapping when browser is not closed and Could have provided option to disable chart in test view and can make visible only in chart.

### ISSUE THREAD

**Issue by Snehaveeranna** [grasshopper7/extentreports-cucumber7-adapter#27]

**Comment by grasshopper7** [comment]
Which report? Can you add screenshots?

**Comment by Snehaveeranna** [comment]
![image](https://user-images.githubusercontent.com/108810588/235842759-9d9f1667-bca9-4b3d-bc0c-d41cf208afb3.png)
Its ExtentHTML report.

**Comment by Snehaveeranna** [comment]
Please let me know if there is any option to resolve this.

**Comment by grasshopper7** [comment]
If you click on the dashboard option again, the chart display will disappear

**Comment by Snehaveeranna** [comment]
Yeah Chart will disappear, but issue is about overlapping. If you see the screenshot testcases view should be displayed on right side of the report and left part should have links to open the test cases. But here all the test cases are overlapped.

**Comment by grasshopper7** [comment]
can you also generate the Spark report for the same test case and add the screenshot?

Also can you add details of 'inspect' from the browser for the extra output?

**Comment by Snehaveeranna** [comment]
![image](https://user-images.githubusercontent.com/108810588/236828026-578bc374-ed87-47fc-b71c-25c8876cf490.png)
In Spark also its overlapping.

**Comment by grasshopper7** [comment]
can you add the inspect element details? 
Best if you can share the report, just the relevant test will be enough.

**Comment by Snehaveeranna** [comment]
To log steps in report which one we should use? Scenario.log or ExtentCucumberAdaptor.addteststep?
